modules/oadp-backing-up-pvs-csi.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+[id="oadp-backing-up-pvs-csi_{context}"]
+= Backing up persistent volumes with Container Snapshot Interface snapshots
+
+You back up persistent volumes with Container Snapshot Interface (CSI) snapshots by creating a `VolumeSnapshotClass` CR.
```

Comment:
s/CR/ custom resource (CR) ? This is the first reference in the module. I think we need to spell it out.